src/app/features/tech-record/components/tech-record-summary/tech-record-summary.component.ts

```diff
@@ -71,40 +66,40 @@ export class TechRecordSummaryComponent implements OnInit {
   trlManufacturerTemplate?: FormNode;
   ddaTemplate?: FormNode;
   reasonForCreation?: FormNode;
+  weightsTemplate?: FormNode;
 
   ngOnInit(): void {
     this.vehicleTemplate();
     this.currentBrakeRecord = this.vehicleTechRecord?.brakes;
-    this.calculateVehicleModel()
+    this.calculateVehicleModel();
   }
 
-  constructor(private store:Store) {}
+  constructor(private store: Store) {}
 
   @Input()
-  set editable(isEditable: boolean){
+  set editable(isEditable: boolean) {
     this.isEditable = isEditable;
-    this.calculateVehicleModel()
+    this.calculateVehicleModel();
   }
 
   calculateVehicleModel() {
-    if(this.isEditable) {
+    if (this.isEditable) {
       this.vehicleTechRecordCalculated = cloneDeep(this.vehicleTechRecord!);
       this.vehicleTechRecordCalculated.reasonForCreation = '';
-    }
-    else {
+    } else {
       this.vehicleTechRecordCalculated = this.vehicleTechRecord!;
     }
-    this.store.dispatch(updateEditingTechRecord({techRecord: this.vehicleTechRecordCalculated}));
+    this.store.dispatch(updateEditingTechRecord({ techRecord: this.vehicleTechRecordCalculated }));
   }
 
   // @ts-ignore
```

Comment:
"Can you sort this please?" - Tom Webb talking about his own work, 2022, colourised